usaspending_api/etl/transaction_loaders/fpds_loader.py

```diff
@@ -39,7 +37,7 @@ def delete_stale_fpds(date):
     Returns list of awards touched
     """
 
-    detached_award_procurement_ids = get_deleted_fpds_data_from_s3(date)
+    detached_award_procurement_ids = retrieve_deleted_fpds_transactions(start_datetime=date)
```

Comment:
This is a problem.  It is returning a dictionary of `{date: [transaction_id, ..], ...}` which doesn't work with the remainder of the code in this function.

Reply:
darn. I must have lost those edits. Good find

Reply:
See Line 45 after recent commit